pkg/master/master.go

```diff
@@ -564,6 +583,16 @@ func (m *Master) init(c *Config) {
 	requestInfoResolver := &apiserver.APIRequestInfoResolver{util.NewStringSet(strings.TrimPrefix(defaultVersion.Root, "/")), defaultVersion.Mapper}
 	apiserver.InstallServiceErrorHandler(m.handlerContainer, requestInfoResolver, apiVersions)
 
+	if m.exp {
```

Comment:
It would have to be split into two helper functions, one version-specific and one group-specific. That saves 2 lines for the version-specific branch then 2 lines in the group-specific code, but you just add that back somewhere else in the function declarations. Doing it as a single function with some kind of config isn't really any better. It doesn't look like it will bring any benefit.